metadata/com.github.luben/zstd-jni/1.5.2-5/resource-config.json

```diff
@@ -0,0 +1,13 @@
+{
+  "bundles": [],
+  "resources": {
+    "includes": [
+      {
+        "condition": {
+          "typeReachable": "com.github.luben.zstd.util.Native"
+        },
+        "pattern": "\\Qlinux/amd64/libzstd-jni-1.5.2-5.so\\E"
+      }
+    ]
+  }
+}
```

Comment:
@luben @KJP12 
- I generated this set of GraalVM reachability metadata within WSL under Windows 11. Considering that the GRM central warehouse only has a Linux CI environment, this should not affect CI execution. 

- I'm happy to assume that someone will run the unit tests under Windows and MacOS in the future, and fill in that part. Because my Windows is not configured with a development environment, and I don't have a MacBook.